src/service/resources-impl.js

```diff
@@ -1414,11 +1472,24 @@ export class Resource {
     if (!built) {
       return false;
     }
-    this.state_ = ResourceState_.NOT_LAID_OUT;
+
+    if (this.hasBeenMeasured()) {
+      this.state_ = ResourceState_.READY_FOR_LAYOUT;
```

Comment:
Why do you want to revert it? Without it, you can't calculate positioning in viewport for an unbuilt element.